From 4bbcd6d1da27cbcc742e9f1f393090f20fe0d184 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 20 Jul 2022 14:10:05 +0300
Subject: [PATCH] [SW-95811] gaudi2: export security_enable_jtag_if

security_enable_jtag_if() function from arcboot to be
used to enable JTAG when rma_enabled. So updated
exports.elf for gaudi2 with arcboot exported list
modified to include this symbol. This would avoid
duplicate implementation of same function in ppboot.

New exports.elf for gaudi2 generated on relevant gaudi2
arcboot rom branch i.e. 'gaudi2_secureboot_rom_tapeout_release6'

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: Icc716dc2df74d863b7a545a61fefc73c7f6ee2e2
---

diff --git a/arch/gaudi2/arc/exports.elf b/arch/gaudi2/arc/exports.elf
index cd3abf1..20e3f3e 100644
--- a/arch/gaudi2/arc/exports.elf
+++ b/arch/gaudi2/arc/exports.elf
Binary files differ
